### ISSUE TITLE
Questions about ligrec in squidgy

### ISSUE THREAD

**Issue by HelloWorldLTY** [theislab/squidpy#406]
Is it mainly constructed by cellphonedB? Is the result of this method same as cellphonedB result?

If I intend to find the number of significant pairs, what should I do? Thanks.

**Comment by michalk8** [comment]
Hi @ChineseBest 

> Is it mainly constructed by cellphonedB?

We use our reimplementation of the cellphoneDB algorithm and get the receptor-ligand pairs from Omnipath (though you can
also specify cellphoneDB as the only resource).

>  Is the result of this method same as cellphonedB result?

The best way to answer this is forr you to take a look at this thread (particularly the comment I'm linking): https://github.com/theislab/squidpy/issues/308#issuecomment-812473384
and have a look at the reproducibility notebook [here](https://github.com/theislab/squidpy_reproducibility/blob/master/notebooks/reproducibility/cellphonedb.ipynb).
In short, the p-values correlate highly with the original cellphoneDB implementation and the means should all match.

> If I intend to find the number of significant pairs, what should I do?

Please have a look at our example here: https://squidpy.readthedocs.io/en/stable/auto_examples/graph/compute_ligrec.html

Hope this answers your questions!

**Comment by ChineseBest** [comment]
Thanks for your answer. In fact, I wonder if there are more ways to analyze cell-cell interaction without using cell type label as cluster keywords. Thanks.
